packages/web/docs/src/pages/docs/gateway/other-features/_meta.ts

```diff
@@ -4,5 +4,5 @@ export default {
   security: 'Security',
   'header-propagation': 'Header Propagation',
   testing: 'Testing & Debugging',
-  'custom-plugins': 'Custom Plugins',
+  plugins: 'Plugins',
```

Comment:
fyi, this will cause 404. add redirect or keep the same URL

Reply:
I'd prefer to have it as custom plugins because this page explains how to write a plugin and how the hooks work.

Reply:
so let's revert it, it will also eliminate the 404

Reply:
Reverted